### ISSUE TITLE
Design: FE for event page using more than one image

### ISSUE THREAD

**Issue by xiongjaneg** [department-of-veterans-affairs/va.gov-cms#17254]
## Issue Description
VAMC editors can add a single image to events. Images are not required but are displayed in the Events list if included. The Secretary has emphasized storytelling, and VAMC editors frequently request the ability to add more than one image to events. 

## Problem(s) to solve 
- How might we enable use of more than one image while ensuring an accessible and equitable experience for all Veterans?
    - Note: The editorial experience for embedding is not ideal and likely needs a comprehensive review and improvement. Does it need to be done before we can enable additional photo use? Related: Enhance editorial experience for embedding photos into stories department-of-veterans-affairs/va.gov-team#13906 
    - Amanda and Danielle ([see comment on #15718](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/15718#issuecomment-1800561775)) recommend a special pilot experiment to test capability for support of one main image and one small supporting image on VAMC Events and Stories only. This pilot would be monitored closely for issues/feedback and could be turned off if problems are discovered. This issue covers the design piece only - separate issues will handle editor guidance, investigation of Drupal for options to address file size compression and support for alt text

## Tasks
Consider impact/change and update design (as necessary) for (1) individual VAMC events, (2) event lists, and (3) featured events considering the following
 - Events will be limited to one main image and one small supporting image
   - Images are optional 
   - Editors may choose to use one or more than one image for a given event page
   - What is the experience on the event page if if more than one image has been specified? How does the experience compare to a event which specifies only one image?
      -  Example: [Cincinnati event page](https://www.va.gov/cincinnati-health-care/events/63790/)
   - What is the experience on the event listing page if more than one image has been specified for a given event?
      - Example: [Cincinnati event listing page](https://www.va.gov/cincinnati-health-care/events/)
   - What is the experience for a featured event if more than one image has been specified? 
      - Example: [Featured event on system home page](https://www.va.gov/cincinnati-health-care/)

Note: we'll also need to view and design the editorial experience - this likely needs a separate issue

## Acceptance criteria
- [ ]

**Comment by FranECross** [comment]
@xiongjaneg Please let me know if there's anything you need from me on this item. Thanks!

**Comment by xiongjaneg** [comment]
@FranECross Thanks for asking. This may be Q2 or Q3 in alignment with multiple images for VAMC Stories, so we'll note to align with Public Websites before starting.
